File: week1/Can-Yavuz(a.k.a=TuvaletPompasi)/validPalindrome.js
/**
 * @param {string} s
 * @return {boolean}
 */
var isPalindrome = function (s) {
  const cleanString = s.replace(/[^A-Za-z0-9]/g, '').toLowerCase();
  const reversedString = cleanString.split('').reverse().join('');
  return cleanString === reversedString;
};

File: week1/Erenimo3442/valid_palindrome.py
import re 

class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()
        for str in s:
            if re.search('[^a-zA-Z0-9]', str):
                s = s.replace(str, '')
        if s[::-1] == s:
            return True